cfgov/unprocessed/js/modules/util/Behavior.js

```diff
@@ -0,0 +1,34 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var Behavior = {};
+
+/**
+ * @param {string} selector Used to query dom for elements.
+ * @param {string} event Event type to add to element.
+ * @param {function} eventHandler Callback for event.
+ */
+Behavior.create = function create( selector, event, eventHandler ) {
+  var behaviorElements = document.querySelectorAll( selector );
+  if( behaviorElements.length === 0 ) {
+    return;
+  }
+  for( var i = 0, len = behaviorElements.length; i < len; i++ ) {
+    behaviorElements[i].addEventListener( event, eventHandler, false );
+  }
+};
+
+/**
+ * @param {HTMLNode} behaviorElement Element in which to remove the event.
+ * @param {string} event Event type to remove from the element.
+ * @param {function} eventHandler Callback for event.
+ */
+Behavior.destroy = function destroy( behaviorElement, event, eventHandler ) {
+  behaviorElement.removeEventListener( event, eventHandler );
+};
+
+// Expose public methods.
+module.exports = Behavior;
```

Comment:
This is implying `Behavior` is a class for `new Behavior()` instances, which is not how it's used. `Behavior` should be updated to use a constructor or update this to:
```
module.exports = {
  create: create,
  destroy: destroy
}
```

And downcase the `behavior.js` filename.